docs/data/base/components/slider/slider.md

```diff
@@ -174,6 +174,17 @@ To let users set the start and end of a range on a slider, provide an array of v
 
 {{"demo": "RangeSlider.js"}}
 
+### Value label
+
+The value label can be rendered over the thumb by using the optional `slots` prop with the `valueLabel` slot. The value label can either be:
```

Comment:
```suggestion
The value label can be rendered around the thumb by using the optional `slots` prop with the `valueLabel` slot. There are typically two use-cases of when the value label is shown:
```

over => round Why? Because MUI Base should not have an opinion how something is rendered